docs/admin/administration.md

```diff
@@ -94,38 +94,60 @@ INFO, DEBUG.
 
 ## System Monitoring
 
-DAOS maintains a set of metrics on I/O and internal state. Each DAOS server can
-be configured to provide an HTTP endpoint for metrics collection. This
-endpoint presents the data in a format compatible with
-[Prometheus](https://prometheus.io).
+The DAOS servers maintain a set of metrics on I/O and internal state
+of the DAOS processes. The metrics collection is very lightweight and
+is always enabled. It cannot be manually enabled or disabled.
+
+The DAOS metrics can be accessed locally on each DAOS server,
+or remotely by configuring an HTTP endpoint on each server.
+
+### Local metrics collection with daos\_metrics
+
+The `daos-server` package includes the `daos_metrics` command-line tool.
+This tool fetches metrics from the local host only.
+No configuration is required to use the `daos_metric` command.
+
+By default, `daos_metrics` displays the metrics in a human-readable tree format.
+To produce CSV formatted output, use `daos_metrics --csv`.
+
+Each DAOS engine maintains its own metrics.
+The `--srv_idx` parameter can be used to specify which engine to query, if there
+are multiple engines configured per server.
+The default is to query the first engine on the server (index 0).
 
-### Configuration
+See `daos_metrics -h` for details on how to filter metrics.
+
+### Configuring the servers for remote metrics collection
+
+Each DAOS server can be configured to provide an HTTP endpoint for metrics
+collection. This endpoint presents the data in a format compatible with
+[Prometheus](https://prometheus.io).
 
-To enable remote telemetry collection, update your DAOS server configuration
-file:
+To enable remote telemetry collection, update the control plane section of
+your DAOS server configuration file:
 
 ```
 telemetry_port: 9191
 ```
 
-The default port number is 9191. Each control plane server will present its
-local metrics via the endpoint: `http://<host>:<port>/metrics`
+By default, the HTTP endpoint is disabled. The default port number is 9191,
+and it is recommended to use this port as it is also the default for the
+clients that will collect the metrics.  Each control plane server will present
+its local metrics via the endpoint: `http://<host>:<port>/metrics`
 
-### Collection
+### Remote metrics collection with dmg telemetry
```

Comment:
Good cleanup. While we're at it, I think it might be helpful to clearly note in this section that it uses the same HTTP endpoint as the prometheus client.